### DISCUSSION TITLE
Fusermount3: mount failed: Permission denied

### DISCUSSION THREAD

**Discussion by abhishekrukhaiyar** [I-am-PUID-0/DMB#95] [Q&A]
Hi,

I'm getting this error when starting the container. 

`INFO - rclone mount name "DMB": 2024/12/31 18:04:08 mount helper error: fusermount3: mount failed: Permission denied
CRITICAL - rclone mount name "DMB": Fatal error: failed to mount FUSE fs: fusermount: exit status 1`

Afterward, this error is periodically coming.

`ERROR - The Riven frontend process is not running. | The Riven backend process is not running. | The Rclone RD process for DMB is not running.`

Any idea why this is happening?

I'm on Ubuntu 24.04. Using Traefik+Portainer.

Let me know if additional information is required.

**Comment by I-am-PUID-0** [comment]
Do you have the below items set in the compose?

```
    devices:
      - /dev/fuse:/dev/fuse:rwm
    cap_add:
      - SYS_ADMIN
```

**Reply by abhishekrukhaiyar** [reply]
Yeah,

It is there.

After searching a lot. The solution was to reset Fuse using the following command.
`sudo apt-get -y purge fuse3`
